### PR TITLE
coreclr: Automatic port of triple slash from System.Text.Unicode

### PR DESCRIPTION
**Pull request by carlossanlop**
Area owners @GrabYourPitchforks, @layomia, @tarekgh please make sure the descriptions are correct. After I check-in this PR, please feel free to start manually documenting the rest of the APIs that didn't have any triple slash comments that could be automatically ported.
Adding @mairaw @rpetrusha for language review.

### CODE REVIEW

File: xml/System.Text.Unicode/Utf8.xml [hunk 1]
```diff
@@ -41,15 +41,29 @@
...
+        <param name="replaceInvalidSequences"><see langword="true" /> to replace invalid UTF-16 sequences in <paramref name="source" /> with U+FFFD; <see langword="false" /> to return <see cref="F:System.Buffers.OperationStatus.InvalidData" /> if invalid characters are found in <paramref name="source" />.</param>
+        <param name="isFinalBlock"><see langword="true" /> if the method should not return <see cref="F:System.Buffers.OperationStatus.InvalidData" />; otherwise, <see langword="false" />.</param>
+        <summary>Converts a UTF-16 character span to a UTF-8 encoded byte span.</summary>
+        <returns>A value that indicates that status of the conversion.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+### Remarks
+
+This method corresponds to the [UTFEncoding.GetBytes](xref:System.Text.UTF8Encoding.GetBytes%2A) method, except that it has a different calling convention, different error handling mechanisms, and different performance characteristics.
```
Comment (carlossanlop) [inline xml/System.Text.Unicode/Utf8.xml]: The formatting used for this GetBytes xref is unusual. @rpetrusha can you explain why you added it that way?

Comment (rpetrusha) [reply xml/System.Text.Unicode/Utf8.xml]: I hate to admit it, @carlossanlop, but I can't think of why I might have done that. It allows you to link to customize the text used to describe a type/member or assign a type/member name other than the one you're linking to (such as when you want to describe an inherited member as if it belong to the derived class), but I can't think of why I used it here. It build successfully, in any case.

Comment (carlossanlop) [reply xml/System.Text.Unicode/Utf8.xml]: As long as it compiles and looks good, it's fine.

File: xml/System.Text.Unicode/Utf8.xml [hunk 2]
```diff
@@ -41,15 +41,29 @@
...
-        <param name="isFinalBlock">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">A UTF-16 encoded read-only character span.</param>
+        <param name="destination">A UTF-8 encoded byte span.</param>
+        <param name="charsRead">When the method returns, the number of characters read from <paramref name="source" />.</param>
+        <param name="bytesWritten">When the method returns, the number of characters written to <paramref name="destination" />.</param>
+        <param name="replaceInvalidSequences"><see langword="true" /> to replace invalid UTF-16 sequences in <paramref name="source" /> with U+FFFD; <see langword="false" /> to return <see cref="F:System.Buffers.OperationStatus.InvalidData" /> if invalid characters are found in <paramref name="source" />.</param>
+        <param name="isFinalBlock"><see langword="true" /> if the method should not return <see cref="F:System.Buffers.OperationStatus.InvalidData" />; otherwise, <see langword="false" />.</param>
```
Comment (ahsonkhan) [inline xml/System.Text.Unicode/Utf8.xml]: @rpetrusha - this documentation is inaccurate. I have submitted a fix for it.

https://github.com/dotnet/coreclr/blob/d4ab484be11b54eb92204cd084638a03a8f9c441/src/System.Private.CoreLib/shared/System/Text/Unicode/Utf8.cs#L30-L31

See https://github.com/dotnet/dotnet-api-docs/pull/3208

**Pull request merged**